src/pages/authentication/Signup.jsx

```diff
@@ -0,0 +1,111 @@
+import { Link } from "react-router-dom";
+import { CheckboxInput, FormInput } from "../../components";
+import { useState } from "react";
+import { validateForm } from "./FormValidator";
+// import { signupuser } from "../../../api/apicall";
+import { useNavigate } from "react-router";
+import { signupuser } from "../../api/apicalls";
+
+const initialSignUpFormState = {
+	name: "",
+	email: "",
+	password: "",
+	confirm_password: "",
+	accept_terms: false,
+};
+const Signup = () => {
+	const [signUpFormState, setsignUpFormState] = useState(
+		initialSignUpFormState
+	);
+	const [formError, setFormError] = useState(initialSignUpFormState);
+	let navigate = useNavigate();
+
+	const inputChangeHandler = (e) => {
+		const new_val = { [e.target.name]: e.target.value };
+		setsignUpFormState((prev) => ({ ...prev, ...new_val }));
+	};
+	const checkboxChangeHandler = (e) => {
+		setsignUpFormState((prev) => ({
```

Comment:
suggestion: setsignUpFormState should be like ------> setSignUpFormState

Reply:
changed function name to setSignUpFormState